index.bs

```diff
@@ -49,198 +49,207 @@ conversion domain.
 
-<dl dfn-for="conversion">
-: <dfn>conversion source</dfn>
+<dl dfn-for="attribution trigger">
+: <dfn>trigger origin</dfn>
 :: An [=url/origin=].
-: <dfn>conversion data</dfn>
+: <dfn>trigger data</dfn>
 :: A [=string=].
-: <dfn>conversion time</dfn>
+: <dfn>trigger time</dfn>
 :: A point in time.
+: <dfn>reporting endpoint</dfn>
+:: An [=url/origin=].
 
 </dl>
 
-<h3 dfn-type=dfn>Conversion report</h3>
+<h3 dfn-type=dfn>Attribution report</h3>
 
-A conversion report is a [=struct=] with the following items:
+An attribution report is a [=struct=] with the following items:
 
-<dl dfn-for="conversion report">
-: <dfn>impression data</dfn>
+<dl dfn-for="attribution report">
+: <dfn>event id</dfn>
 :: A [=string=].
-: <dfn>conversion data</dfn>
+: <dfn>trigger data</dfn>
 :: A [=string=].
-: <dfn>attribution credit</dfn>
+: <dfn>credit</dfn>
 :: An integer in the range [0, 100].
 
 </dl>
 
 # Algorithms # {#algorithms}
 
-<h3 algorithm id="parsing-conversion-destination">Parsing a conversion destination</h3>
+<h3 algorithm id="parsing-attribution-destination">Parsing an attribution destination</h3>
 
-To <dfn>parse a conversion destination</dfn> from an <{a}> tag |anchor|,
+To <dfn>parse an attribution destination</dfn> from a string |str|:
 1. Let |url| be the result of running the [=URL parser=] on the value of
-    the |anchor|'s <{a/conversiondestination}>.
+    the |str|.
 1. Return the result of [=obtain a site|obtaining a site=] from |url|'s
     [=url/origin=].
 
-<h3 algorithm id="creating-impression">Activating an impression</h3>
+<h3 algorithm id="obtaining-attribution-source-anchor">Obtaining an attribution source from an <code>a</code> element</h3>
 
-To <dfn>activate an impression</dfn> from an <{a}> tag |anchor|,
+To <dfn>obtain an attribution source</dfn> from an <{a}> element |anchor|:
 1. Let |currentTime| be the current time.
-1. Let |impression| be a new [=impression=] struct whose items are:
+1. Let |source| be a new [=attribution source=] struct whose items are:
 
-    : [=impression/impression source=]
+    : [=attribution source/source origin=]
     :: |anchor|'s [=relevant settings object=]'s [=environment/top-level origin=].
-    : [=impression/impression data=]
-    :: The result of applying [=parsing conversion data=] to |anchor|'s
-        <{a/impressiondata}> attribute.
-    : [=impression/conversion destination=]
-    :: The result of running [=parse a conversion destination=] on |anchor|.
-    : [=impression/reporting endpoint=]
-    :: The [=url/origin=] of the result of running the [=URL parser=] on the value
-        of |anchor|'s <{a/reportingorigin}> attribute.
-    : [=impression/expiry=]
-    :: |currentTime| +  <{a/impressionexpiry}> milliseconds.
-    : [=impression/impression time=]
+    : [=attribution source/event id=]
+    :: The result of running [=parse attribution data=] with |anchor|'s
+        <{a/attributionsourceeventid}> attribute and [=max event id value=].
```

Comment:
Here I'd probably phrase it 
```suggestion
        <{a/attributionsourceeventid}> attribute modulo [=max event id value=].
```

and match that in the definition. It doesn't quite match the advice in https://infra.spec.whatwg.org/#algorithm-params, but I think it winds up more readable.